enfivy-back/src/main/java/com/envify/back/controller/PackageVersionController.java

```diff
@@ -53,6 +57,13 @@ public ResponseEntity<PackageVersionEntity> findPackageVersionById(@PathVariable
         return ResponseEntity.ok().body(packageVersionEntity);
     }
 
+    @GetMapping("/{id}/config_files")
+    public ResponseEntity<ConfigPackageFileEntity> findPackageConfigFiles(@PathVariable int id) {
+        ConfigPackageFileEntity configPackageFileEntity = configPackageFileService.findConfigPakageFileByPakageVersionId(id);
```

Comment:
findConfigPakageFileByPakageVersionId => findConfigPackageFileByPackageVersionId

(miss the 'c' for package)